api/src/main/java/jakarta/data/repository/Find.java

```diff
@@ -27,48 +27,49 @@
 
 /**
  * <p>Annotates a repository method as a parameter-based automatic query method
- * that returns entities or entity attributes.</p>
+ * that returns entity instances or entity attribute values.</p>
  *
  * <p>The {@code Find} annotation indicates that the annotated repository
- * method executes a query to retrieve entities (or a subset of entity attributes)
- * based on its parameters and the arguments assigned to those parameters.</p>
+ * method executes a query against a certain entity type, filtering records
+ * based on its parameters and the arguments assigned to its parameters.</p>
  *
- * <p>The method return type identifies one of the following:</p>
  * <ul>
- * <li>the entity type returned by the query,</li>
- * <li>a single entity attribute type returned by the query (requires the
- *     {@link Select} annotation), or</li>
- * <li>a Java record type representing a subset of entity attributes returned
- *     by the query. The names of the record components and entity attributes
- *     must match, or the entity attribute names must be specified by the
- *     {@link Select} annotation.</li>
+ * <li>If the method return involves an entity type, according to the rules
+ *     specified below, the return type determines the queried entity type.</li>
+ * <li>Otherwise, if the method return type does not involve an entity type,
+ *     the queried entity type is the {@linkplain #value entity class explicitly
+ *     specified by the annotation}, if any, or the primary entity type of the
+ *     repository.</li>
  * </ul>
  *
```

Comment:
This paragraph was lost during the rewrite.  This was added to ensure it is implementable based on Jakarta Persistence/JPQL. (At the time it was written the JDQL for this pattern didn't exist yet, but JDQL needs to remain a subset of JPQL and cannot go beyond it or become incompatible with it)
```suggestion
 * <p>When returning a single entity attribute or a subset of entity attributes,
 * the attribute types must be single-valued basic types, not multiple-valued
 * types such as collections, arrays, or associations.</p>
 *
```

Reply:
It wasn't lost. It was changed to:

```java
 * <p>When the return type is not the queried entity type, every returned
 * attribute must be a basic attribute.</p>
 ```

which uses a term with a precise definition in the spec.

Reply:
> It wasn't lost. It was changed to:
> 
> ```java
>  * <p>When the return type is not the queried entity type, every returned
>  * attribute must be a basic attribute.</p>
> ```
> 
> which uses a term with a precise definition in the spec.

I didn't notice that because it also moved to a different a place. This is fine. No issue here.